src/interface/trace.ts

```diff
@@ -54,19 +54,25 @@ export enum InterfaceElementName {
   AUTOROUTER_VISUALIZATION_ROW = 'expandable-autorouter-visualization-row',
   BLOG_LINK = 'blog-link',
   CAREERS_LINK = 'careers-link',
+  CLOSE_BUTTON = 'close-button',
   COMMON_BASES_CURRENCY_BUTTON = 'common-bases-currency-button',
   CONFIRM_SWAP_BUTTON = 'confirm-swap-or-send',
   CONNECT_WALLET_BUTTON = 'connect-wallet-button',
   CONTINUE_BUTTON = 'continue-button',
   DISCONNECT_WALLET_BUTTON = 'disconnect-wallet-button',
   DOCS_LINK = 'docs-link',
+  DOWNLOAD_BUTTON = 'download-button',
   EXPLORE_BANNER = 'explore-banner',
   EXPLORE_POOLS_TAB = 'explore-pools-tab',
   EXPLORE_SEARCH_INPUT = 'explore_search_input',
   EXPLORE_TOKENS_TAB = 'explore-tokens-tab',
   EXPLORE_TRANSACTIONS_TAB = 'explore-transactions-tab',
+  EXTENSION_DOWNLOAD_BUTTON = 'extension-download-button',
+  EXTENSION_DOWNLOAD_CONNECTOR = 'extension-download-connector',
+  EXTENSION_LAUNCH_PROMOTIONAL_MODAL = 'extension-launch-promotional-modal',
   FIAT_ON_RAMP_BUY_BUTTON = 'fiat-on-ramp-buy-button',
   FIAT_ON_RAMP_LEARN_MORE_LINK = 'fiat-on-ramp-learn-more-link',
+  GETTING_STARTED_MODAL = 'GETTING_STARTED_MODAL',
```

Comment:
these two can go to `InterfaceModalName`! Also matching formatting on `GETTING_STARTED_MODAL` to the rest!